diff -Naur a/game-core/src/main/java/games/strategy/engine/ClientFileSystemHelper.java b/game-core/src/main/java/games/strategy/engine/ClientFileSystemHelper.java
--- a/game-core/src/main/java/games/strategy/engine/ClientFileSystemHelper.java	2018-11-16 18:26:37.000000000 -0600
+++ b/game-core/src/main/java/games/strategy/engine/ClientFileSystemHelper.java	2020-05-04 19:55:22.656711789 -0500
@@ -97,8 +97,8 @@
    */
   public static File getUserRootFolder() {
     final File userHome = new File(SystemProperties.getUserHome());
-    final File rootDir = new File(new File(userHome, "Documents"), "triplea");
-    return rootDir.exists() ? rootDir : new File(userHome, "triplea");
+    final File rootDir = new File(new File(userHome, "Documents"), ".triplea");
+    return rootDir.exists() ? rootDir : new File(userHome, ".triplea");
   }
 
   /**
